### ISSUE TITLE
Client crash with 3.6.1.0 can't update to 3.6.2.1

### ISSUE THREAD

**Issue by moabie** [OreCruncher/DynamicSurroundings#611]
**Mod Version: 3.6.1.0**
**Forge Version: 1.12.2 14.23.5.2854**
**Link to crash log: 
[crash-2020-11-22_22-11.txt](https://github.com/OreCruncher/DynamicSurroundings/files/5580392/crash-2020-11-22_22-11.txt)
**

**Description:**
Yesterday I installed forge 1.12.2 (client + server) + 35mods 
was fine for a couple of hours, 1 client had game crashes
today next 2 clients have crashes, including me; same crash report (as in file)

I tried to update to ver. 3.6.2.1, 3.6.2.0 - unable: server doesn't start, frozen at line: "[CSRV] Executing Java..."
when i delete mod (in newest ver.), server goes on, no issues

maybe conflict with 'animania' mod because short crash message shown by Minecraft says:
"The game crashed whilst rendering entity in world
Error: java.lang.NoClassDefFoundError: com/animania/common/entities/sheep/EntityEweBase
Ending code: -1"

**Comment by OreCruncher** [comment]
See https://github.com/OreCruncher/DynamicSurroundings/issues/573.  Make sure you install the other Animania mods because of cross dependency.

**Comment by moabie** [comment]
What i've done: 
I tried do downgrade DS to ver. 3.6.0.2 - no changes
I uninstalled DS - no changes (!) so it doesn't look like these 2 mods dependencies (at least this issue)
I uninstalled animania-extra mod (animania-base and animania-farm left installed) - it's OK!

So i lost a few animals (including beautiful peacock), but I still have most of them and have DS, so I don't have to resign any of them :)

ps. still don't know why sheep according error (com/animania/common/entities/sheep/EntityEweBase) appeared while crash..?
